schema/resolvers.js

```diff
@@ -119,7 +119,42 @@ const Resolvers = {
     deleteOption: (_, args, ctx) =>
       ctx.repositories.Option.remove(args.input.id),
     undeleteOption: (_, args, ctx) =>
-      ctx.repositories.Option.undelete(args.input.id)
+      ctx.repositories.Option.undelete(args.input.id),
+    createOtherAnswer: async (root, args, ctx) => {
+      const parentAnswer = await ctx.repositories.Answer.get(
+        args.input.parentAnswerId
+      );
+
+      if (!isNil(parentAnswer.otherAnswerId)) {
+        return parentAnswer;
+      }
+
+      if (parentAnswer.type === "Radio" || parentAnswer.type === "Checkbox") {
```

Comment:
Again, adding to anything other than these should be an error i think?

Reply:
Also, you could add these as a `where` clause when fetching? Let the DB do the work.